src/modules/user/routes/index.js

```diff
@@ -1,11 +1,12 @@
 // user routes here
 const express = require('express');
+const { authenticate } = require('../../../middlewares/authenticate');
 
 const router = express.Router();
 
 const {
-  registerUser, loginUser, initateResetPassword, resetPassword,
-  getUserById, updateUserById, verifyUser
+  registerUser, loginUser, getUserById,
+  updateUserById, verifyUser, logout, initateResetPassword, resetPassword
```

Comment:
You split this line in another one probably put (initateResetPassword, resetPassword) on a new line to prevent a lint errors that might happen.

Reply:
addressed